simpleline/render/screen_stack.py

```diff
@@ -0,0 +1,76 @@
+# Screen stack classes for manipulating stack which are in a view stack.
+#
+# Copyright (C) 2017  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Author(s): Jiri Konecny <jkonecny@redhat.com>
+#
+
+
+from simpleline import SimplelineError
+
+
+class ScreenStackException(SimplelineError):
+    pass
+
+
+class ScreenStackEmptyException(ScreenStackException):
+    pass
+
+
+class ScreenStack(object):
+
+    def __init__(self):
+        self._screens = []
+
+    def is_empty(self):
```

Comment:
Yeah, this has a compatibility reason. When you will look on the python-3 containers like `Queue`. You will see that these are methods not properties.

https://docs.python.org/3/library/queue.html#queue.Queue.empty

Of course we don't need to inherit this behavior but it looks to me more consistent to have it like python 3 library more than like Anaconda project. However, it should be renamed to `empty()`, instead of `is_empty()`.